### PR TITLE
code cleaned up a bit

### PR DESCRIPTION
**Pull request by corybenton**
- deleted console logs
- deleted excess spaces
- added some spaces for ease of reading
- got rid of the if null lines
- cleaned up for loop for disliking liked breweries
- added removing markers from map when brewery disliked
- cleaned up removing markers from map when removed from favorites
- added comments for functions

### CODE REVIEW

**Review by elmiquea — COMMENTED** [review]
Great.

File: script2.js
Comment (Conartisttt) [inline script2.js:1]: Script 2 favorites page seems to no longer display "You have no liked breweries"

**Comment by corybenton** [comment]
my bad missed that one. should be fixed now

**Review by Conartisttt — APPROVED** [review]
LGTM

**Comment by corybenton** [comment]
why were the lines on 236-238 not accepted?

**Pull request merged**